src/main/java/io/iworkflow/controller/ShortlistCandidatesController.java

```diff
@@ -0,0 +1,170 @@
+package io.iworkflow.controller;
+
+import io.iworkflow.core.Client;
+import io.iworkflow.core.ClientSideException;
+import io.iworkflow.gen.models.ErrorSubStatus;
+import io.iworkflow.workflow.shortlistcandidates.EmployerOptInWorkflow;
+import io.iworkflow.workflow.shortlistcandidates.ShortlistWorkflow;
+import io.iworkflow.workflow.shortlistcandidates.model.ImmutableEmployerOptInInput;
+import io.iworkflow.workflow.shortlistcandidates.model.ImmutableShortlistInput;
+import io.iworkflow.workflow.shortlistcandidates.utils.WorkflowUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Map;
+
+
+@Controller
+@RequestMapping("/shortlist_candidates")
+public class ShortlistCandidatesController {
+
+    @Autowired
+    private Client client;
+
+    @PostMapping("/opt_in")
+    public ResponseEntity<String> optIn(
+            @RequestBody Map<String, Object> requestBody
+    ) {
+        final String employerId = requestBody.get("employerId").toString();
+
+        final String workflowId = WorkflowUtil.buildEmployerOptInWorkflowId(employerId);
+
+        final ImmutableEmployerOptInInput input = ImmutableEmployerOptInInput.builder()
+                .employerId(employerId)
+                .build();
+
+        try {
+            // The timeout is set to 0, indicating that the workflow will never time out
+            client.startWorkflow(EmployerOptInWorkflow.class, workflowId, 0, input);
+        } catch (final ClientSideException e) {
+            if (e.getErrorSubStatus() == ErrorSubStatus.WORKFLOW_ALREADY_STARTED_SUB_STATUS) {
+                return ResponseEntity.ok(String.format("Employer %s has already opted in", employerId));
+            }
+            throw new RuntimeException(String.format("optIn failed with workflow %s", workflowId), e);
+        }
+
+        return ResponseEntity.ok(String.format("Started workflowId: %s", workflowId));
+    }
+
+    @PostMapping("/opt_out")
+    public ResponseEntity<String> optOut(
+            @RequestBody Map<String, Object> requestBody
+    ) {
+        final String employerId = requestBody.get("employerId").toString();
+
+        final String workflowId = WorkflowUtil.buildEmployerOptInWorkflowId(employerId);
+
+        final EmployerOptInWorkflow rpcStub = client.newRpcStub(
+                EmployerOptInWorkflow.class,
+                workflowId,
+                ""
```

Comment:
Lol This reminds me to create a new issue to improve: https://github.com/indeedeng/iwf-java-sdk/issues/new
Not CR feedback.